### PR TITLE
Check file headers in CircleCI

### PR DESCRIPTION
**Pull request by matthew-white**
I noticed that a few new files are missing file headers. I think it's easy to miss those, so I wrote a small script to check that in CircleCI.

#### What has been done to verify that this works as intended?

The first version of the script didn't specify anything for `skip`. For each file that the script listed, I either added a file header to the file or added the file to `skip`. I verified that when the script lists files, it exits with an exit code of 1.

#### Why is this the best possible solution? Were any other approaches considered?

We do other checks in CircleCI, so this seemed like a potentially useful addition.

#### How does this change affect users? Describe intentional changes to behavior and behavior that could have accidentally been affected by code changes. In other words, what are the regression risks?

Not user-facing.

#### Does this change require updates to the API documentation? If so, please update docs/api.md as part of this PR.

No change required.

#### Before submitting this PR, please make sure you have:

- [x] run `make test-full` and confirmed all checks still pass OR confirm CircleCI build passes
- [x] verified that any code from external sources are properly credited in comments

### CODE REVIEW

File: lib/external/sentry.js [hunk 1]
```diff
@@ -1,3 +1,12 @@
+// Copyright 2018 ODK Central Developers
```
Comment (matthew-white) [inline lib/external/sentry.js:1]: It looks like this file was created in 2018: 3d1064060455bc60a193ad003bcde83d7d7ff392. (The file was moved in eca4b52e8eebe52f949ae0fda632d226402f439c.)

File: lib/bin/check-file-headers.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+// This might not be the full list in local development: .gitignore includes
+// other files. However, this list should be sufficient for CircleCI.
+const skip = new Set([
+  '.circleci',
+  '.git',
+  '.github',
+  'config',
+  'docs',
+  'lib/bin/.eslintrc.js',
+  'lib/util/quarantine/pkcs7.js',
```
Comment (matthew-white) [inline lib/bin/check-file-headers.js:31]: @issa-tseng, do you know whether this file needs a file header?

File: lib/bin/check-file-headers.js [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+  '.eslintrc.json',
+  '.gitignore',
+  '.npmrc',
+  'CONTRIBUTING.md',
+  'LICENSE',
+  'Makefile',
+  'NOTICE',
+  'README.md',
+  'package-lock.json',
+  'package.json'
```
Comment (matthew-white) [inline lib/bin/check-file-headers.js:43]: The last time I checked, it wasn't possible to add comments in package.json.

Comment (matthew-white) [inline lib/bin/check-file-headers.js:43]: @lognaturel, let me know if you think any of these files needs a file header!

**Comment by matthew-white** [comment]
@alxndrsn, tagging you for review in case you have time to take a look!

**Comment by matthew-white** [comment]
@ktuite and I are taking a look now! This PR may conflict with #577, so I'm thinking we should try to merge this first.

**Review by ktuite — APPROVED** [review]

**Comment by matthew-white** [comment]
I'm going to go ahead and merge, since I think this is an improvement, and I want to be able to resolve any conflicts with #577. @lognaturel and @issa-tseng, let me know if there are any other files that you think I should add a file header to!

File: lib/bin/check-file-headers.js (hunk 3 above)
Comment (alxndrsn) [reply lib/bin/check-file-headers.js:43]: I wouldn't expect any JSON file to allow comments.  In the current form, the header is probably only applicable to `.js` files

Comment (matthew-white) [reply lib/bin/check-file-headers.js:43]: Comments in a JSON file would definitely be nonstandard (though I feel like I've seen it somewhere? 🤔). And agreed about the current header only really working for .js files. I'll probably add a similar script to Frontend at some point. But Frontend has .js and .vue files with slightly different headers, so the script will have to account for that somehow.

**Pull request merged**